Api/Src/Services/GradeService.ts

```diff
@@ -0,0 +1,27 @@
+import * as mongoose from "mongoose";
+
+import GradeRepository from "../Repositories/GradeRepository";
+import { GradeType } from "../Models/Grade";
+
+export default class CourseService {
```

Comment:
CourseService? 🙈

Reply:
haha spotted, I inspired  myself with some clean and smart solutions